### ISSUE TITLE
After entering "<img src>", Boostnote shows a blank window

### ISSUE THREAD

**Issue by gmqi** [BoostIO/Boostnote#3217]
# Current behavior

The note's content as following:
``` 
 ![High tones](https://cms.buzzi.space/api/attachments/3259?width=900)
 <img src>
```

After entering "`<img src>`", Boostnote shows a blank windon.
 I close Boostnote and open it again, this issue still exist, I could not use Boostnote anymore.

After I delete "`<img src>`" or add a blank line before it, this issue disappear.
[](url)

# Expected behavior
Boostnote should work normally for any text contents.

# Steps to reproduce

1. Enter a link: ` ![High tones](https://cms.buzzi.space/api/attachments/3259?width=900)`
2. Enter ` <img src>`
3. Boostnote shows a blank screen

# Environment

- Version : 0.12.1
- OS Version and name : Win 7 / 10

**Comment by keelejak** [comment]
Same problem occurs after typing `<a href="">`.

**Comment by hikerpig** [comment]
Same as #3159, already fixed by a47dac285408
